.babelrc

```diff
@@ -3,10 +3,10 @@
     [
       "env",
       {
-        "loose": true
+        "loose": true,
+        "modules": false,
       }
     ],
-    "stage-0",
```

Comment:
stage-0 wasn't necessary, I would recommend you to avoiding relying on stage-* presets. Use a [plugins](https://babeljs.io/docs/plugins/#transform-plugins-es2015) directly.